annotations.md

```diff
@@ -40,6 +40,13 @@ This specification defines the following annotation keys, intended for but not l
     ```
 * **org.opencontainers.image.title** Human-readable title of the image (string)
 * **org.opencontainers.image.description** Human-readable description of the software packaged in the image (string)
+* **org.opencontainers.image.base.digest** [Digest](descriptor.md#digests) of the image this image is based on (string)
+  * This SHOULD be the immediate image sharing zero-indexed layers with the image, such as from a Dockerfile `FROM` statement.
+  * This SHOULD NOT reference any other images used to generate the contents of the image (e.g., multi-stage Dockerfile builds).
+* **org.opencontainers.image.base.ref.name** Image reference of the image this image is based on (string)
```

Comment:
Same example here. I assume we're saying this is the fully qualified location of where the descriptor was found.
`registry.example.com/repo/image:tag`
with no expectation for default registries, meaning, we do not support an assumed docker hub value if we see hello-world:latest. It must be `docker.io/hello-world:latest`

Reply:
I think this came up in the meeting where this was introduced, and I think I recall people being okay with supporting any form of reference according to the grammar in reference.go, with no attempt at canonicalizing `ubuntu` -> `index.docker.io/library/ubuntu`.

If that sounds okay to you I can add an example to clarify that it's just any string the user provides, it SHOULD be parseable according to reference.go, and it SHOULD be a reference to an image where the base image was originally found, and might be able to be found now.

Reply:
For individual companies, they of course will always do whatever they need for their teams workflow. 
For software vendors/projects who are publishing content, what is the expectation? If we don't have a defined standard, how will tooling work consistently against this value?

Reply:
The ref.name is still a bit confusing on the grammar. I was reading this as it would follow a fully qualified artifact, including the registry name. Basically, a docker pull should resolve, (without assuming a default registry). That doesn't mean the user has access, but it does identify where the base ref sourced from.

Reply:
I think that's a reasonable request. I didn't realize that users could configure a "default registry" aside from Dockerhub anyway, so TIL, and with that it makes sense to say it SHOULD be the fully qualified ref name, to avoid ambiguity.

Reply:
This should reference the ref.name grammar.